### PR TITLE
Make Stanford Profile Great Again

### PR DESCRIPTION
**Pull request by sherakama**
# Make it better 2.0!

### CODE REVIEW

**Comment by pookmish** [comment]
judgement

**Comment by sherakama** [comment]
Just wait for the tests to pass 😆

**Comment by pookmish** [comment]
AUTO MERGE!

**Comment by pookmish** [comment]
:rofl: failed tests!

**Comment by sherakama** [comment]
Blast!

**Comment by jbickar** [comment]
troll

**Comment by imonroe** [comment]
So does this get copied over to SOE or what?

File: composer.json [hunk 1]
```diff
@@ -100,6 +100,7 @@
         "composer/installers": "^1.2",
         "drupal/address": "^1.2",
         "drupal/allowed_formats": "^1.1",
+        "drupal/bad_judgement": "^2.0",
         "drupal/block_content_permissions": "^1.8",
```
Comment (pookmish) [inline composer.json:104]:
```suggestion
        "drupal/block_content_permissions": "^1.8",
        "drupal/thanos": "^1.0",
```

**Review by pookmish — CHANGES_REQUESTED** [review]

**Pull request closed without merging**